src/openrct2/rct2/RCT2.h

```diff
@@ -260,7 +261,7 @@ namespace OpenRCT2::RCT2
         uint8_t unreliabilityFactor; // 0x198
         // Range from [0, 100]
         uint8_t downtime;                                      // 0x199
-        uint8_t inspectionInterval;                            // 0x19A
+        RideInspection inspectionInterval;                     // 0x19A
```

Comment:
Revert. In general you should be using the uint8_t type for the rct2/1 headers and you should be validating the conversion into RideInspection to ensure invalid values aren't there. (although saying that we don't normally actually do the validation)